config/routes.rb

```diff
@@ -58,6 +58,11 @@
 
   root 'v0/example#index', module: 'v0'
 
+  # route for testing with ID.me locally without front-end vets-website repo
+  if Rails.env.development?
+    get '/auth/login/callback', to: 'v0/sessions#saml_callback', module: 'v0'
```

Comment:
My point being that ID.me is going to 302 the browser to localhost:3001/auth/login/callback which will still produce a 404 because the vets-api runs on :3000